transports/transport-rest/transport-rest-vertx/src/main/java/org/apache/servicecomb/transport/rest/vertx/VertxRestDispatcher.java

```diff
@@ -182,6 +182,8 @@ private void onRequest(RoutingContext context) {
 
     RestProducerInvocation restProducerInvocation = new RestProducerInvocation();
     context.put(RestConst.REST_PRODUCER_INVOCATION, restProducerInvocation);
+    restProducerInvocation.setAfterCreateInvocationHandler(
```

Comment:
both edge and normal need to set this, so set it to be default action?

Reply:
The code has been refactored, please review the new commit